react-native/react/util/setNotifications.js

```diff
@@ -14,15 +15,30 @@ let channelsSet = {}
 export default function (channels: NotificationChannels): Promise<void> {
   return new Promise((resolve, reject) => {
     channelsSet = {...channelsSet, ...channels}
+
+    const toSend = {
```

Comment:
so this function is a bit strange. we can turn stuff on, but never off really. this is very brittle. So incoming channels have to be true, with a subset of keys. we pass all the keys in, as they're required, but missing ones will be false